src/behaviours/pod-logs/download-logs.test.tsx

```diff
@@ -41,14 +54,16 @@ describe("download logs options in pod logs dock tab", () => {
     builder.setEnvironmentToClusterFrame();
 
     callForLogsMock = jest.fn();
+    getLogsMock = jest.fn();
+    getSplittedLogsMock = jest.fn();
 
     builder.beforeWindowStart((windowDi) => {
       windowDi.override(callForLogsInjectable, () => callForLogsMock);
 
       // Overriding internals of logsViewModelInjectable
-      windowDi.override(getLogsInjectable, () => () => ["some-logs"]);
-      windowDi.override(getLogsWithoutTimestampsInjectable, () => () => ["some-logs"]);
-      windowDi.override(getTimestampSplitLogsInjectable, () => () => [...logs]);
```

Comment:
I think it would be better if we overrode `callForLogs` (which is the dependency of `logStore`) instead. That way we can have better confidence that the business logic within LogStore is correct.